src/Audit/Drupal/SolrVersion.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drutiny\Audit\Drupal;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\Annotation\Param;
+
+/**
+ * Check the version of Drupal project in a site.
+ * @Param(
+ *  name = "module",
+ *  description = "The module to version information for",
+ *  type = "string"
+ * )
+ * @Param(
+ *  name = "version",
+ *  description = "The static version to check against.",
+ *  type = "string"
+ * )
+ * )
+ */
+class SolrVersion extends Audit
+{
+    /**
+     * @inheritdoc
+     */
+    public function configure()
+    {
+        $this->addParameter(
+            'module',
+            static::PARAMETER_OPTIONAL,
+            'Module to check.'
+        );
+        $this->addParameter(
+            'version',
+            static::PARAMETER_OPTIONAL,
+            'Module to check.'
+        );
+    }
+
+    public function audit(Sandbox $sandbox)
+    {
+        $module = $sandbox->getParameter('module');
+        // Version of SOLR we need to find if exists.
+        $version_searched = $sandbox->getParameter('version');
+
+        $info = $sandbox->drush(['format' => 'json'])->pmList();
+        $module_version = $info[$module]['version'];
+
+        // Do we have version 3 or 4?
+        preg_match('/^2.*.*/', $module_version, $solr34_found);
```

Comment:
Customers could still be using 8.x-1.x acquia_search module, which would qualify as Solr3/4.

Reply:
I'm thinking this wont cover D7 ... which is a different beast because the same module version can use both Solr 3/4 and Solr 7...

Reply:
would be good to see different examples, so we can find what would be the best logic to add here. Right now I'm only testing this with a customer in D8 and SOLR 4.

Where can we see examples of SOLR 7 customers that we can use for testing this?

Reply:
If this is Acquia specific logic, then the audit class should really live in https://github.com/drutiny/plugin-acquia